`UnsupportedFeatureException: An object of type 'org.apache.pdfbox.rendering.ImageType$2' was found in the image heap`

Native image build fails with 
```
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: An object of type 'org.apache.pdfbox.rendering.ImageType$2' was found in the image heap. This type, however, is marked for initialization at image run time for the following reason: classes are initialized at run time by default.
```

More info: https://github.com/quarkiverse/quarkus-tika/actions/runs/6794028039/job/18469822743

--- a/runtime/src/main/java/io/quarkus/tika/runtime/graal/TikaFeature.java
+++ b/runtime/src/main/java/io/quarkus/tika/runtime/graal/TikaFeature.java
@@ -9,6 +9,7 @@ public class TikaFeature implements Feature {
         final String reason = "Quarkus run time init for Apache Tika";
         RuntimeClassInitialization.initializeAtRunTime("org.apache.pdfbox.pdmodel", reason);
         RuntimeClassInitialization.initializeAtRunTime("org.apache.pdfbox.rendering", reason);
+        RuntimeClassInitialization.initializeAtBuildTime(org.apache.pdfbox.rendering.ImageType.class, reason);
     }
 
     @Override

--- a/runtime/src/main/java/io/quarkus/tika/runtime/graal/TikaFeature.java
+++ b/runtime/src/main/java/io/quarkus/tika/runtime/graal/TikaFeature.java
@@ -9,7 +9,7 @@ public class TikaFeature implements Feature {
         final String reason = "Quarkus run time init for Apache Tika";
         RuntimeClassInitialization.initializeAtRunTime("org.apache.pdfbox.pdmodel", reason);
         RuntimeClassInitialization.initializeAtRunTime("org.apache.pdfbox.rendering", reason);
-        RuntimeClassInitialization.initializeAtBuildTime(org.apache.pdfbox.rendering.ImageType.class, reason);
+        RuntimeClassInitialization.initializeAtBuildTime(org.apache.pdfbox.rendering.ImageType.class);
     }
 
     @Override

